client/src/js/components/map.jsx

```diff
@@ -39,7 +39,17 @@ var Map = React.createClass({
       }
     ).addTo(map);
 
-    L.geoJson(items).addTo(map);
+    var marker = new 
+    L.Icon({
+      iconUrl: "images/marker-icon.png"
+    });
+
+    function content (feature,layer) {
+      layer.bindPopup("<h1>" + feature.properties.Title + "</h1>");
+      layer.setIcon(marker);
+    };
+
```

Comment:
I prefer to put function declarations towards the bottom because they will get hoisted.  This would be a good opportunity to learn about hoisting and the difference between function declarations and function expressions